pysus/online_data/SINAN.py

```diff
@@ -95,12 +96,17 @@ def download(year, disease, cache=True):
     ftp.login()
     ftp.cwd(path)
     try:
-        df = _fetch_file(fname, path, 'DBC')
+        _fetch_file(fname, path, 'DBC', return_df=False)
     except:  # If file is not part of the final releases
-        df = _fetch_file(fname, path_pre, 'DBC')
-
+        _fetch_file(fname, path_pre, 'DBC', return_df=False)
+    if return_fname:
```

Comment:
when is return_fname defined true  ?
Its value is always false?

Reply:
yes, its default value is False.
When the user wants to download a very large SINAN file,it can pass `return_fname=True` so that the data is not load into memory to return a Dataframe. Instead a file name is returned to the user.

Reply:
After the dbf to csv conversion is done, wouldn't it be nice to return the dataframe instead of the filename string to the get_chunked function?

>%time df = download(year=2017,disease='zika', return_fname=True) 

return of *get_chunked_dataframe()* with **pd.read_csv()**
```
CPU times: user 2.77 s, sys: 87.9 ms, total: 2.86 s
Wall time: 9.45 s
```  
return the filename string
``` 
CPU times: user 2.73 s, sys: 40.6 ms, total: 2.77 s
Wall time: 9.37 s
```